pennylane/labs/tests/vibrational_ham/test_bosonic.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit Tests for the Boseonic representation classes."""
+import pickle
+from copy import copy, deepcopy
+
+import numpy as np
+import pytest
+from scipy import sparse
+
+import pennylane as qml
+from pennylane import numpy as pnp
+from pennylane.labs.vibrational_ham.bosonic import (
+    BoseWord,
+    BoseSentence,
+)
+
+
+class TestBoseWord:
+    # Expected bose sentences were computed manually or with openfermion
```

Comment:
Is this correct? OpenFermion was used to generate the references?

Reply:
Yes, you can use OpenFermion to generate the references since they have their own version of `normal_order()`. It is a little different from ours in terms of ordering (they order based on indices due to the commutation rules of bosons) but their results are equivalent.

Reply:
To confirm, you have used OpenFermion to obtain these reference rsults?

Reply:
The first one I did manually by hand, the second I used openFermion.